backend/src/main/resources/application-dev.properties

```diff
@@ -1,3 +1,11 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/inventorsoftacademy?useLegacyDatetimeCode=false&serverTimezone=UTC
 spring.datasource.username=scripty
-spring.datasource.password=scripty
\ No newline at end of file
+spring.datasource.password=scripty
+
+api.title=Scripty API Documentation
```

Comment:
All those properties won't change between environments(except urls). Therefore we can move them into generic `application.properties` config file